src/npc.cpp

```diff
@@ -104,6 +104,36 @@ npc::npc()
     last_updated = calendar::turn;
 }
 
+standard_npc::standard_npc( const std::string &name, const std::vector<itype_id> &clothing,
+                            int sk_lvl, int s_str, int s_dex, int s_int, int s_per )
+{
+    this->name = name;
+    position = { 0, 0, 0 };
```

Comment:
> It isn't magic, except for the part where it is outside the allowed range.

I'd say it's rather magic since its meaning isn't obvious. It would be better to simply use a named `tripoing` constant (`invalid_tripoint` would work rather well for the purpose) and check `!inbounds(p)` before attempting to access an invalid item.